### ISSUE TITLE
Make `w` and `h` optional and use `image.naturalWidth` and `image.naturalHeight`

### ISSUE THREAD

**Issue by moldcraft** [dimsemenov/PhotoSwipe#957]
http://www.w3schools.com/jsref/prop_img_naturalwidth.asp

**Comment by dimsemenov** [comment]
These properties can be read only after image is loaded, PhotoSwipe needs dimensions right away.

Duplicate of https://github.com/dimsemenov/PhotoSwipe/issues/741

**Comment by pedrovagner** [comment]
Duplicated https://github.com/dimsemenov/PhotoSwipe/issues/664
